analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/SymbolicExecution/SETestContext.cs

```diff
@@ -42,31 +42,30 @@ public SETestContext(string code, AnalyzerLanguage language, SymbolicCheck[] add
         }
 
         public static SETestContext CreateCS(string methodBody, params SymbolicCheck[] additionalChecks) =>
-            CreateCS(methodBody, null, additionalChecks);
+            CreateCS(methodBody, null, null, additionalChecks);
 
-        public static SETestContext CreateCS(string methodBody, string additionalParameters, params SymbolicCheck[] additionalChecks)
+        public static SETestContext CreateCS(string methodBody, string additionalParameters, params SymbolicCheck[] additionalChecks) =>
+            CreateCS(methodBody, additionalParameters, null, additionalChecks);
+
+        public static SETestContext CreateCS(string methodBody, string additionalParameters, string localFunctionName, params SymbolicCheck[] additionalChecks)
         {
             var code = $@"
 using System;
 using System.Collections.Generic;
-
 public class Sample
 {{
     public static int StaticField;
     public static int StaticProperty {{get; set;}}
     public int Property {{get; set;}}
     private int field;
-
     private bool Condition => Environment.ProcessorCount == 42;  // Something that cannot have constraint
-
```

Comment:
This empty line should not be removed. It separates state (fields and properties) and important method.